apps/cyberstorm-storybook/stories/components/PaginationButton.stories.tsx

```diff
@@ -0,0 +1,17 @@
+import { ComponentStory, ComponentMeta } from "@storybook/react";
+import { PaginationButton } from "@thunderstore/cyberstorm";
+import React from "react";
+
+const meta = {
+  title: "Cyberstorm/Components/PaginationButton",
+  component: PaginationButton,
+} as ComponentMeta<typeof PaginationButton>;
+
+const Template: ComponentStory<typeof PaginationButton> = (args) => (
+  <PaginationButton {...args} />
+);
+
+const DefaultButton = Template.bind({});
```

Comment:
This was more of @VilppeRiskidev vision how we should do this. Argument was that it's better documentation of button itself and shows how the actual it works and what arguments could be given.

Reply:
Yup, I think (change my mind) that the storybook's main purpose is documentation. I think having a story for each component documents how to use them. At least it shows the props.
One thing to do (in the "near" future = I'd say right after or even during my current milestone), would be to order the components into their own folders, which I'd also relfect to the storybook.

Reply:
I agree with the idea that documenting the components in Storybook is something we should pursue.

However, on this particular component's case, I disagree.

* The component will probably never be used just by itself, only within the Pagination component (at least the name implies so), so it's documented in Pagination component's story
* There's currently exactly one story. When checking the code, I see the component takes no props nor children. If this is supposed to be documentation, it's IMO quite poor documentation. It tells me nothing about the how the component can be used.
* Visually this renders an empty button that's barely visible on the background. Again, if this is supposed to be documentation, we're currently documenting a use case no one will probably ever want to use

That being said, I acknowledge that this is a low impact issue. However, I think it might work as precedent and affect the shape of our storybook in the future. So even if nothing gets done, I'd like to say for the record that IMO it would be better to either have the story document some viable use cases, or remove the story altogether,